content/en/blog/_posts/2022-04-28-Increasing-the-security-bar-in-Ingress-NGINX/index.md

```diff
@@ -45,7 +45,7 @@ Linux containers (and underlying mechanisms such as kernel namespaces) work.
 You can read about cgroups in the Kubernetes glossary: [`cgroup`](https://kubernetes.io/docs/reference/glossary/?fundamental=true#term-cgroup) and learn more about cgroups interact with namespaces in the NGINX project article
 [What Are Namespaces and cgroups, and How Do They Work?](https://www.nginx.com/blog/what-are-namespaces-cgroups-how-do-they-work/).
 (As you read that, bear in mind that Linux kernel namespaces are a different thing from
-[Kubernetes namespaces](h/docs/concepts/overview/working-with-objects/namespaces/)).
+[Kubernetes namespaces](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/)).
```

Comment:
@Tej-Singh-Rana, we do not use direct hyperlinking as a standard within the K8s project website. Please could you amend this to the below?
```suggestion
[Kubernetes namespaces](/docs/concepts/overview/working-with-objects/namespaces/)).
```

Reply:
Thanks, didn't know about this. 
Can you please check again? I changed it. 

Regards,